packages/payment-processor/src/payment/utils-near.ts

```diff
@@ -44,6 +44,7 @@ export const isNearAccountSolvent = (
     const fungibleContract = new Contract(nearWalletConnection.account(), token.value, {
       changeMethods: [],
       viewMethods: ['ft_balance_of'],
+      useLocalViewExecution: true,
```

Comment:
This parameter was added in https://github.com/near/near-api-js/pull/1172. I don't fully understand it, but I don't think it will break anything. It might make view functions to the same blockId more efficient.